www/includes/easyparliament/hansardlist.php

```diff
@@ -2757,6 +2757,126 @@ public function biggest_debates($args=array()) {
         return $this->_get_data_by_biggest_debates($args);
     }
 
+    public function _get_data_by_featured_gid($args=array()) {
```

Comment:
I did think about that but there's enough differences that factoring it out would require lots of conditionals which I think would make the code harder rather than easier to read.